prow/config.yaml

```diff
@@ -505,7 +505,7 @@ presubmits:
         - "--service-account=/etc/service-account/service-account.json"
         - "--upload=gs://kubernetes-jenkins/pr-logs"
         - "--" # end bootstrap args, scenario args below
-        - "--test=//... //hack:verify-all -//build/... -//vendor/... -//examples/..."
+        - "--test=//... //hack:verify-all -//build/... -//vendor/..."
```

Comment:
I think `//hack:verify-all` needs to be a `--manual-test` now?

Reply:
yes. I should query the other branches for manual as well.

Reply:
moved `//hack:verify-all` for each branch, I think test is fine now, inspecting build.